replication_importing_with_backup.sh

```diff
@@ -73,9 +53,9 @@ if [[ $exit_code_imp_1 != 0 && $exit_code_imp_1 != 137 ]] || \
     [[ $exit_code_imp_2 != 0 && $exit_code_imp_2 != 137 ]] || \
     [[ $exit_code_bck != 0 && $exit_code_bck != 137 ]]; then
   echo "Could not import/backup"
-  shutdown
-  exit 1
+  # shutdown
+  # exit 1
 fi
 
 echo "Passed!"
-shutdown
+# shutdown
```

Comment:
why commented out?

Reply:
good catch, that was me testing locally something  🤦🏼‍♂️